src/shrutiC/CodingExam4.java

```diff
@@ -0,0 +1,44 @@
+package shrutiC;
+
+public class CodingExam4 {
+
+	void findSimilarNumInArray(int[] array1, int[] array2) {
+
+		String similarNum = "";
+
+		for (int i = 0; i < array1.length; i++) {
+
+			if (array1[i] == array2[i])
+				similarNum = similarNum + " " + Integer.toString(array1[i]);
```

Comment:
Hi Shruti,
you need to compare element in two loop, fortunately both array have same common value on same index, but think on diff index,
Pls update accordingly. I know this is not mentioned but this is like counter question on same program.